app/helpers/admin/topical_event_featurings_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Admin::TopicalEventFeaturingsHelper
```

Comment:
Took me a while to piece together what this helper function was doing. Just wondering if there would be a way to do this more at the model layer rather than the view layer?

Reply:
great shout. Will sort this